app/models/manageiq/providers/amazon/storage_manager/s3.rb

```diff
@@ -38,4 +38,9 @@ def connect(options = {})
     options[:service] = :S3
     parent_manager.connect options
   end
+
+  def connect_regional(region)
+    @regional_connection ||= {}
+    @regional_connection[region] ||= connect(:region => region)
```

Comment:
Not sure I like that we are caching the connection.  Is there a reason for that?

Additionally, this feels like it should just be a parameter to connect method above instead of a whole method.  That is, the caller can just do `connect(:region => region)`